salt/modules/purefa.py

```diff
@@ -0,0 +1,1274 @@
+# -*- coding: utf-8 -*-
+
+##
+# Copyright 2017 Pure Storage Inc
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
There is no need to separately license this file. The entire project is licensed under Apache 2.0

Reply:
@cachedout same response as above